spec/bright_serializer/serializer_params_spec.rb

```diff
@@ -44,5 +44,21 @@ def upcase(object, params)
     it 'serialize params' do
       expect(instance.to_hash).to eq(result)
     end
+
+    context 'when not passing params' do
+      # The anonymous serializer use params in a way that it take for granted that params is always a hash
+      # and call params[:suffix], so if params is nil it will raise an error
+      # This test ensure that when params is not passed it will be an empty hash and no error is raised.
```

Comment:
Peut être que quelque chose comme ça serait plus claire comme commentaire : 
```suggestion
# The anonymous serializer expects params to always be a hash and accesses params[:suffix]. 
      # If params is nil, this would previously raise an error. 
      # This test ensures that when params is not provided, an empty hash is passed instead, preventing any error.
```

Reply:
J'ai pas les commentaires qui parle au passé, "previously raise an error". C'est bon pour un change log, commit message, mais quand on regarde le code je trouve cela porte à confusion d'avoir un commentaire qui parle d'une autre version du code qu'on lit.

```suggestion
      # The anonymous serializer assumes params is always a hash and calls params[:suffix] directly.
      # This test ensures that when params is not passed, it is treated as an empty hash without raising an error.
```

Reply:
C'est jusque il était inscrit "so if params is nil it will raise an error". J'avais l'impression que tu disais que avant, ça aurait raise une erreur. Mais avec le "will" ça portait à confusion.

Reply:
Ta raison 🤔 bien vu fallait corrigé ça, merci!